### ISSUE TITLE
Changing docker container name

### ISSUE THREAD

**Issue by amir20** [amir20/dozzle#2558]
Since we are starting to make changes to containers, I think it would be really cool if double clicking on a container would allow someone to change the name of it.

**Comment by amir20** [comment]
@akash-ramaswamy do you know if this possible? I wonder if anybody would use it.

**Comment by akash-ramaswamy** [comment]
Yes @amir20 . I think they have it in their sdk! and we need to call it through the `actions`
https://github.com/docker/docker/blob/v24.0.7/client/container_rename.go#L9

This is a cool thing to try out, but is there any real world scenario this could apply? like any use-case?

**Comment by akash-ramaswamy** [comment]
If people are using `dockerfile` to spin up their containers, this would be useful. But everyone mostly uses `docker-compose`!

**Comment by amir20** [comment]
My use case was exactly that. I often start a quick container but then realize I didn't give it a name. So then I have to shut it down and do it all over again. 

I think you maybe right though that it won't get a lot of usage. 

Currently, the `--enable-controls` is being used less than 1%. I assume because people are still reading the docs or don't know about it. 

I'll provide more analytics as I collect 'em.

**Comment by akash-ramaswamy** [comment]
Hi @amir20 , I get a feel that this feature would be useful! (*maybe, because this is cool* 😎)

> Currently, the `--enable-controls` is being used less than 1%. I assume because people are still reading the docs or don't know about it.

But we also need to think of this point which you mentioned 🤔

**Comment by ab623** [comment]
I don't think that a name rename as described would be that useful. Or there's minimal use cases where it would apply to. You've mentioned one but are there any others?

Just for reference I also use docker-compose to. Most people using dozzle would too I assume.

**Comment by amir20** [comment]
Yea I think you are right. We can probably close since when I think about it, I also use compose when I really want to deploy something permanently.
